test/draw.py

```diff
@@ -56,49 +33,68 @@ class TestDraw(unittest.TestCase):
 
     # Result image for draw
     def test_draw(self):
-        image = Image.new(imageMode, imageSize, imageBackground)
-        canvas = ImageDraw.Draw(image)
 
-        for i in range(len(points)):
-            draw(canvas, (points[0], points[1]), 'A')
+        """ Create the file name and its saving path, and specify the reference file to compare to."""
+        image_name = filename(sys._getframe().f_code.co_name)
+        result_file, reference_file = get_path(image_name)
+
+        ''' This function is to create an empty image with a specific dimension
+            with white background, and black/white colored '''
+
+        image, canvas = get_image('L', (15,90),'white')
+
+        for i in range(len(draw_points) - 1):
+            draw(canvas, (draw_points[i + 0], draw_points[i + 1]), 'A')
 
-        fileName = sys._getframe().f_code.co_name + '.png'
+        """ saving the file and closing it """
 
-        image.save(outputPath+"/"+fileName, fileType)
+        image.save(result_file)
         image.close()
 
-        resultFile = outputPath + "/" + fileName
-        referenceFile = referencePath + "/" + fileName
+        """ validate the resultant file against the reference images"""
 
-        # compare results agains reference data
-        with open(resultFile, "rb") as result:
-            with open(referenceFile, "rb") as reference:
-                self.assertTrue(reference.read() == result.read())
+        validate(reference_file, result_file)
 
     # Result image for drawWire
     def test_drawWire(self):
-        image = Image.new(imageMode, imageSize, imageBackground)
-        canvas = ImageDraw.Draw(image)
 
-        drawWire(canvas, points)
+        """ Create the file name and its saving path, and specify the reference file to compare to."""
```

Comment:
function help string format should have the """ separator alone on it's own line:

"""
This function ...
"""